packages/dev/core/src/Engines/renderTargetWrapper.ts

```diff
@@ -133,9 +134,11 @@ export class RenderTargetWrapper {
             return value;
         }
 
-        return this._isMulti
+        const result = this._isMulti
             ? this._engine.updateMultipleRenderTargetTextureSampleCount(this, value, initializeBuffers)
             : this._engine.updateRenderTargetTextureSampleCount(this, value);
+        this._samples = value;

```

Comment:
I don't think I should, because since the MSAA renderbuffer is set at the texture level, there can be a discrepancy between the textures sample count. 
The only way to harmonize all the sample counts is to change the `_samples` value of the wrapper, and since there is a cache test, changing this value with `setTexture` of index 0 would early return the call to `setSamples`, leaving the other textures with the wrong sample count

Reply:
I don't think I should, because since the MSAA renderbuffer is set at the texture level, there can be a discrepancy between the textures sample count. 
The only way to harmonize all the sample counts is to change the `_samples` value of the wrapper, and since there is a cache test, changing this value with `setTexture` of index 0 would early return the call to `setSamples`, leaving the other textures with the wrong sample count